src/constants/provinces.ts

```diff
@@ -0,0 +1,15 @@
+export const PROVINCES = [
+  "Alberta",
+  "British Columbia",
+  "Manitoba",
+  "New Brunswick",
+  "Newfoundland and Labrador",
+  "Northwest Territories",
+  "Nova Scotia",
+  "Nunavut",
+  "Ontario",
+  "Prince Edward Island",
+  "Quebec",
+  "Saskatchewan",
+  "Yukon",
+];
```

Comment:
NIT
```suggestion
] as const;
```

It also might be better to just have this as a `JSON` file.

Reply:
Dropdown does not allow passing readonly types to `options`.